### ISSUE TITLE
translation to top of screen

### ISSUE THREAD

**Issue by jolywho** [killkimno/MORT#44]
in some games, like Yakuza 3, the translation cannot be displayed at the top of the game screen

**Comment by killkimno** [comment]
Yakuza is always covering the translation window because it is set to Top Most
Please use the layer translation window. And the translation window should be placed outside the game.

**Comment by AdrianGomezEscalante** [comment]
Hi, I'm having the same problem with Yakuza 4. The program doesn't overlay the translation over the game. But I don't understand the solution you provided. Are you saying there's no way to have the translation appear over the game? Would I need to display it on a second monitor, for example? I only use one monitor.

**Comment by killkimno** [comment]
https://learn.microsoft.com/ko-kr/windows/powertoys/
You mean this program, I'll have to test it out and include it in the guide

**Comment by AdrianGomezEscalante** [comment]
That's right, I'm referring to that program. Running the game in windowed mode didn't work with Yakuza 3, 4, and 5. It seems like the window in those games always takes priority. But using PowerToys worked perfectly for me, and it solved the problem.

It would be great if you could add it to the guide to fix this problem with other video games where it occurs and let people know the solution.

If you need help figuring out how to make it work, let me know.

Thank you very much.
